jdaviz/configs/default/plugins/export/export.vue

```diff
@@ -152,7 +171,7 @@
              :results_isolated_to_plugin="true"
              @click="interrupt_recording"
              :disabled="!movie_recording"
-          >
+          traitlets.String>

```

Comment:
is this an intentional addition?  What does it do?

Reply:
nope this was a mistake, removed